src/components/ebay-notice/index.marko

```diff
@@ -26,7 +26,7 @@ $ var isLightGuidance = isSection && !input.status;
         input.fillWindow && `${type}-notice--fill`,
         input.class
     ]>
-    <if(!isLightGuidance)>
+    <if(!isLightGuidance && !input.iconHidden)>
```

Comment:
I would actually move this or wrap the `ebay-icon` with this. As is if the dialog has a header it won't be shown with `icon-hidden` option

Reply:
Was this comment resolved?

Reply:
yes now ebay-icon blocks are wrapped individually rather than the whole header